examples/asr/quantization/speech_to_text_quant_infer.py

```diff
@@ -23,7 +23,7 @@
 import torch
 from omegaconf import open_dict
 
-from nemo.collections.asr.metrics.wer import WER, word_error_rate
+from nemo.collections.asr.metrics.wer import WER, CTCCharDecoding, CTCCharDecodingConfig, word_error_rate
```

Comment:
Looks like the equivalent class for CTCCharDecoding is RNNTDecoding, right? Shouldn't we make them consistent by adding Char to the RNNT class or drop char here?

Reply:
Let's drop Char. Bad idea on my part.

Reply:
Done.